### PR TITLE
Feature/confirmation notification

### PR DESCRIPTION
**Pull request by aatauil**

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -63,3 +64,40 @@ def process_send_notifications():
...
+
+        
+def send_confirmation_notifications():
+    """
+    Check for messages in berichtencentrum received from kalliope and send email to dossierbehandelaar letting them know the email has arrived succesfully
+
+    :returns ?:
+    """
+    helpers.log("fetching messages that need a confirmation notification to be sent ...")
+    q = find_kalliope_mail(PUBLIC_GRAPH, 70)
```
Comment (claire-lovisa) [inline tasks.py:76]: Shouldn't the `70` be configurable as an environment variable ?

File: tasks.py [hunk 2]
```diff
@@ -63,3 +64,40 @@ def process_send_notifications():
...
+      here = os.path.dirname(os.path.realpath(__file__)) 
+      templatepath = os.path.join(here, 'templates/confirmatie_bericht.hbs')
+      with open(templatepath, 'r') as email_template_file:
+          compiler = Compiler()
+          precompiled = compiler.precompile(email_template_file.read())
+          email_html_template = compiler.template(precompiled)
+    for bericht in berichten:
+        subject = "Confirmatie {0}: '{1}' - Uw bericht is succesvol toegekomen in het berichtencentrum van {2}  ".format(bericht['dossiernummer']['value'], bericht['betreft']['value'], bericht['ontvanger']['value'])
+        content = None # NOTE: Not used, we use html_content
+        email = new_email("noreply@test.test", bericht['mailadres']['value'], subject, content)
```
Comment (claire-lovisa) [inline tasks.py:89]: Same here, the address should be configurable no ?

File: queries.py [hunk 3]
```diff
@@ -149,3 +149,38 @@ def construct_mail_query(graph_uri, email, outbox_folder_uri):
...
+                    ext:heeftBehandelaar ?behandelaar.
+
+                ?behandelaar schema:email ?mailadres.
+            }}
+          ?van skos:prefLabel ?verzender.
+          ?naar skos:prefLabel ?ontvanger.
+          FILTER (?ontvangDatum > "{0}"^^xsd:dateTime)
+        }}
+        ORDER BY DESC(?verzonden)
+        LIMIT 200
```
Comment (claire-lovisa) [inline queries.py:184]: We miss a filter to ensure we don't process the same message multiple times

**Review by claire-lovisa — CHANGES_REQUESTED** [review]
See comments but we are missing a filter & a few env variables :)

File: tasks.py (hunk 1 above)
Comment (aatauil) [reply tasks.py]: Yess Replaced this by MAX_AGE env

File: tasks.py (hunk 2 above)
Comment (aatauil) [reply tasks.py]: Yes this should be the noreply email similar to regular notifications so replaced that by the correct ENV

File: queries.py (hunk 3 above)
Comment (aatauil) [reply queries.py]: Filter added + graph matcher that similar to regular notification query

File: tasks.py (hunk 1 above)
Comment (claire-lovisa) [reply tasks.py]: Might be useful to have a different value than the one used for the other notification no ?

**Review by claire-lovisa — APPROVED** [review]
All good, thanks ! :)

**Pull request merged**